Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/new_rule_modal.tsx

```diff
@@ -71,64 +57,60 @@ export class NewRuleButton extends React.Component<Properties, State> {
     })();
     const buttonWrapper = (() => {
       if(this.props.displaySize === DisplaySize.SMALL) {
-        return NewRuleButton.STYLE.buttonWrapperSmall;
+        return NewRuleModal.STYLE.buttonWrapperSmall;
       } else {
-        return NewRuleButton.STYLE.buttonWrapper;
+        return NewRuleModal.STYLE.buttonWrapper;
       }
     })();
     const options = [];
     for(let i = 0; i < this.props.schemas.length; ++i) {
       if(i === this.state.selection) {
         options.push(
-          <div className={css(NewRuleButton.EXTRA_STYLE.selectedRow)}
+          <div className={css(NewRuleModal.EXTRA_STYLE.selectedRow)}
               onClick={this.onClickRule.bind(this, i)}>
             {this.props.schemas[i].name}
           </div>);
       } else {
         options.push(
-          <div className={css(NewRuleButton.EXTRA_STYLE.optionRow)}
-            onClick={this.onClickRule.bind(this, i)}>
+          <div className={css(NewRuleModal.EXTRA_STYLE.optionRow)}
+              onClick={this.onClickRule.bind(this, i)}>
             {this.props.schemas[i].name}
           </div>);
       }
     }
     return (
       <div>
-        <div style={NewRuleButton.STYLE.newRuleRow}
+        <div style={NewRuleModal.STYLE.newRuleRow}
             onClick={this.props.onToggleModal}>
-          <div style={NewRuleButton.STYLE.imageWrapper}>
-            <img src='resources/account_page/compliance_page/add.svg'
+          <div style={NewRuleModal.STYLE.imageWrapper}>
+            <img src={NewRuleModal.ADD_PATH}
               height={plusSignImageSize}
               width={plusSignImageSize}/>
           </div>
-          <div style={NewRuleButton.STYLE.newRuleText}>{'Add New Rule'}</div>
+          <div style={NewRuleModal.STYLE.newRuleText}>{'Add New Rule'}</div>
```

Comment:
You can remove the curly brackets from 'Add New Rule', ie.

    <div>Add New Rule</div>